components/addresses/bus/index.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-plusplus */
+import React, { Component } from 'react';
+import {
+  View, Image, Button, Text, Modal
+} from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import shuttleScheduleInformation from '../../home/menu/shuttleBusSchedule/shuttleScheduleService';
+import styles from './styles';
+import bus from './bus.png';
+
+
+export default class Bus extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clicked: false
+    };
+  }
+
+  /**
+ * Returns the time of the next shuttle bus.
+ * @param {string} campus - Either SGW or LOY.
+ * If any other string is passed, by default it will be the SGW campus.
+ *
+ */
+  getNextShuttleBus(campus) {
+    const hours = new Date().getHours();
+    let min = new Date().getMinutes();
+
+    // We know that the shuttle busses only come at times that are divisible by 5.
+    min = (min % 5 !== 0) ? (min - (min % 5) + 5) : min;
+
+    const totalSeconds = (hours * 3600) + (min * 60); // Converting the current time in seconds
```

Comment:
Maybe have a constant variable SECONDS_IN_HOUR for 3600 and MINUTES_IN_HOUR for 60 to improve readability.

Reply:
Agree!